samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _rpl-border-router-sample:
+
+RPL Border Router
+#################
+
+Overview
+********
+
+The RPL border router sample application for Zephyr provides a HTTP(S) server
+and net shell for management purposes. Typically it would be used to connect to
+IEEE 802.15.4 network but Bluetooth IPSP network functionality is also possible.
+
+The source code for this sample application can be found at:
+:file:`samples/net/rpl_border_router`.
+
+Requirements
+************
+
+- Linux machine with web browser and the screen terminal emulator
+- Either QEMU or real device like Freedom Board (FRDM-K64F)
+- For QEMU see this :ref:`networking_with_qemu`
+- Ethernet access for management purposes
+
+Building and Running
+********************
+
+Currently, the HTTP Server application is configured to listen at port 80.
+If you want to modify the http server config options, please check
+the configuration settings in :file:`samples/net/rpl_border_router/src/main.c`
+file and also in the :file:`samples/net/rpl_border_router/src/config.h` file.
+
+To use QEMU for testing, follow the :ref:`networking_with_qemu` guide.
+
+This sample code supports both static and dynamic (DHCPv4) IP addresses that
+can be defined in the project configuration file:
```

Comment:
suggest adding the name of the configuration file:

       in the project configuration file :file:`samples/net/rpl_border_router/prj.conf`: